ui.py

```diff
@@ -558,57 +558,100 @@ def open_reports_window(self):
         Opens a new window to view and print reports
         :return None:
         """
-
-        # Create the pop-up window ------------------------------------
+        # Create the pop-up window for class reports------------------------------------
         report_window: tk.Toplevel = tk.Toplevel(self.admin_frame)
-        report_window.title("Reports")
+        report_window.title("Class Reports")
         report_window.geometry("500x400")
-        # --------------------------------------------------------------
 
-        # Range of dates ----------------------------------------------
-        range_label: ttk.Label = ttk.Label(
-            report_window,
-            text="Specify Date Range",
-            font=("Roboto", 12)
-        )
+        # file name text input
+        class_file_name_label: ttk.Label = ttk.Label(report_window, text="File Name",font=("Roboto", 12))
+        class_file_name_label.pack(pady=5)
+        class_file_name: tk.Entry = tk.Entry(report_window, width=20)
+        class_file_name.pack(pady=5)
 
-        range_label.pack(pady=10)
+        # company name text input
+        class_company_name_label: ttk.Label = ttk.Label(report_window, text="Company Name",font=("Roboto", 12))
+        class_company_name_label.pack(pady=5)
+        class_company_name: tk.Entry = tk.Entry(report_window, width=20)
+        class_company_name.pack(pady=5)
 
-        start_date: tk.Entry = tk.Entry(report_window, width=20)
-        start_date.pack(pady=5)
+        # title text input
+        class_title_label: ttk.Label = ttk.Label(report_window, text="Title Name",font=("Roboto", 12))
+        class_title_label.pack(pady=5)
+        class_title_name: tk.Entry = tk.Entry(report_window, width=20)
+        class_title_name.pack(pady=5)
 
-        end_date: tk.Entry = tk.Entry(report_window, width=20)
-        end_date.pack(pady=5)
-        # ---------------------------------------------------------------
+        # TODO: add calender for the date selection
 
-        # Employee selection --------------------------------------
-        employee_label: ttk.Label = ttk.Label(
+        # Print big report button
+        print_class_button: tk.Button = tk.Button(
             report_window,
-            text="Select Employee",
-            font=("Roboto", 12))
+            text="Print All Users Report",
+            command=lambda: self.print_class_report(class_file_name.get(), class_company_name.get(), class_title_name.get(), "11-11-24", "11-16-24"),
```

Comment:
Make sure start date and end date are automated

Reply:
I'll resolve this by getting rid of the dates entirely for the class reports